### DISCUSSION TITLE
Text color change

### DISCUSSION THREAD

**Discussion by isamuk** [nicufarmache/lovelace-big-slider-card#31] [Q&A]
Hi this is a very handy custom card, is there a way to over-ride the default text color behaviour? I have a custom theme in this case LCARS and the text of the slider come out white when I want black. I suspect there is a hierarchy I have yet to understand but I don't see an option to change that color even with card-mod.

**Comment by nicufarmache** [comment]
I'm going to add an option for text color the next version

**Reply by nicufarmache** [reply]
You can try it now

**Reply by isamuk** [reply]
Excellent thanks!
